internal/util/util.go

```diff
@@ -191,3 +191,40 @@ func GetClients() (*kubernetes.Clientset, *snapshotterClientSet.Clientset, error
 
 	return client, snapshotterClient, nil
 }
+
+// IsVolumeSnapshotClassHasListerSecret returns whether a volumesnapshotclass has a snapshotlister secret
+func IsVolumeSnapshotClassHasListerSecret(vc *snapshotv1beta1api.VolumeSnapshotClass) bool {
+	// https://github.com/kubernetes-csi/external-snapshotter/blob/master/pkg/utils/util.go#L59-L60
+	// There is no release w/ these constants exported. Using the strings for now.
+	_, nameExists := vc.Annotations[PrefixedSnapshotterListSecretNameKey]
+	_, nsExists := vc.Annotations[PrefixedSnapshotterListSecretNamespaceKey]
+	return nameExists && nsExists
+}
+
+// IsVolumeSnapshotContentHasDeleteSecret returns whether a volumesnapshotcontent has a deletesnapshot secret
```

Comment:
Minor nit: Period at the end for valid godoc string.

Reply:
added the `.` in https://github.com/vmware-tanzu/velero-plugin-for-csi/pull/37/commits/e4018c28326f338f6c346f6e8d40502d8c66ae7b